Scripts fixes

fyliu opened the pull request:
Fixes #221

### What changes did you make?

- fixed various script issues
- added ProgramArea tests (to satisfy coverage and provide an example of a GET list test)

### Why did you make the changes (we will use this info to test)?

- the scripts were missing some needed functionality
- some test coverage was missing

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<!-- Note, if your images are too big, use the <img src="" width="" length="" />  syntax instead of ![image](link) to format the images -->

<!-- If images are not loading properly, you might need to double check the syntax or add a newline after the closing </summary> tag -->

<details>
<summary>Visuals before changes are applied</summary>

![image](Paste_Your_Image_Link_Here_After_Attaching_Files)

</details>

<details>
<summary>Visuals after changes are applied</summary>

![image](Paste_Your_Image_Link_Here_After_Attaching_Files)

</details>

alexlaw528 commented:
Here are my results from running the scripts.

Can you verify that there will be discrepancies between the coverage % between running a filtered test and the main `test.sh `script?

I'm getting 78% coverage on the filtered test and 99% on the main one.

Results from running `test.sh -k program_area`
![Screenshot 2023-10-28 at 12 38 16 PM](https://github.com/hackforla/peopledepot/assets/95406690/402048e4-3f53-4173-b478-f5a4f9851dda)

Results from running `tests.sh`
![Screenshot 2023-10-28 at 12 58 07 PM](https://github.com/hackforla/peopledepot/assets/95406690/bb3c5ea8-f696-4b34-81b8-215abd12e3bb)

fyliu commented:
@alexlaw528 yes, selecting a subset of tests to run will affect the coverage number to be lower than running all the tests.

I updated the documentation to describe some of the fixes. For the `-k` option, I added the recommendation to also use the `--no-cov` option to disable the coverage report, since it's not useful in that case.

alexlaw528 reviewed (approved):
Minor suggestion in setup.cfg but otherwise looks good!

fyliu commented on app/setup.cfg:
```diff
@@ -11,4 +11,4 @@ skip_glob = */migrations/*.py
 DJANGO_SETTINGS_MODULE = peopledepot.settings
 python_files = tests.py test_*.py *_tests.py
 # addopts = -vv -x --last-failed --cov --cov-report html
```
Oh, that's commented out because we don't use it. I included the line as some options that might be useful in some cases but not used. It also shows the syntax to use for adding options.
